pkg/apis/v1alpha1/register.go

```diff
@@ -55,23 +55,36 @@ var (
 	AMIFamilyBottlerocket = "Bottlerocket"
 	AMIFamilyAL2          = "AL2"
 	AMIFamilyUbuntu       = "Ubuntu"
+	AMIFamilyWindows2019  = "Windows2019"
+	AMIFamilyWindows2022  = "Windows2022"
 	AMIFamilyCustom       = "Custom"
 	SupportedAMIFamilies  = []string{
 		AMIFamilyBottlerocket,
 		AMIFamilyAL2,
 		AMIFamilyUbuntu,
+		AMIFamilyWindows2019,
+		AMIFamilyWindows2022,
 		AMIFamilyCustom,
 	}
 	SupportedContainerRuntimesByAMIFamily = map[string]sets.String{
 		AMIFamilyBottlerocket: sets.NewString("containerd"),
 		AMIFamilyAL2:          sets.NewString("dockerd", "containerd"),
 		AMIFamilyUbuntu:       sets.NewString("dockerd", "containerd"),
+		AMIFamilyWindows2019:  sets.NewString("dockerd", "containerd"),
+		AMIFamilyWindows2022:  sets.NewString("dockerd", "containerd"),
 	}
+
+	Windows2019                                           = "2019"
+	Windows2022                                           = "2022"
+	WindowsCore                                           = "Core"
+	Windows2019Build                                      = "10.0.17763"
```

Comment:
Just checking: This value isn't defined anywhere in `corev1` in any upstream file. It would be nice if this mapped to an upstream constant, but, if not, that's also fine. Doesn't hurt to check

Reply:
No such const in the upstream project. :( 
Have to add by ourselves.